files/image_config/caclmgrd/caclmgrd

```diff
@@ -225,7 +225,9 @@ class ControlPlaneAclManager(object):
                             rule_cmd = "ip6tables" if table_ip_version == 6 else "iptables"
                             rule_cmd += " -A INPUT -p {}".format(ip_protocol)
 
-                            if "SRC_IP" in rule_props and rule_props["SRC_IP"]:
+                            if "SRC_IPV6" in rule_props and rule_props["SRC_IPV6"]:
```

Comment:
Minor suggestion, but we could simplify these checks a little bit here like so:
```suggestion
                            if rule_props.get("SRC_IPv6"):
```
If the key is not found, then get will return None. Likewise, if the value is None, we'll still get None. This is a pretty minor difference though, so I'll leave it up to you. :)

Reply:
Thanks for the suggestion. I'll keep it in mind when refactoring next. I will stick with this for the time being, as it matches the earlier check; I prefer consistency, so I think all instances should be changed in the same PR.

Reply:
That sounds good to me!